platform/lifecycle_inputs.go

```diff
@@ -85,11 +86,12 @@ func NewLifecycleInputs(platformAPI *api.Version) *LifecycleInputs {
 	inputs := &LifecycleInputs{
 		// Operator config
 
-		LogLevel:    envOrDefault(EnvLogLevel, DefaultLogLevel),
-		PlatformAPI: platformAPI,
-		ExtendKind:  envOrDefault(EnvExtendKind, DefaultExtendKind),
-		UseDaemon:   boolEnv(EnvUseDaemon),
-		UseLayout:   boolEnv(EnvUseLayout),
+		LogLevel:           envOrDefault(EnvLogLevel, DefaultLogLevel),
+		PlatformAPI:        platformAPI,
+		ExtendKind:         envOrDefault(EnvExtendKind, DefaultExtendKind),
+		UseDaemon:          boolEnv(EnvUseDaemon),
+		InsecureRegistries: sliceEnv(EnvInsecureRegistries),
```

Comment:
Maybe we should only read into this field if `platformAPI.AtLeast("0.13")`

Reply:
I wouldn't do that because right now we are not supporting the 0.13 version -the latest is the 0.12- and it would make the `lifecycle_inputs_tests` fail since it's run the test toward the latest version.
I think it can be left as is it since we have the guards in all the steps, it means that the InsecureRegistries field might be populated but never consumed.